build/hyperfine/build.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/bash
+#
+# {{{ CDDL HEADER
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+# }}}
+
+# Copyright 2019 OmniOS Community Edition (OmniOSce) Association.
+# Copyright 2020 Stephen Gregoratto
+
+. ../../lib/functions.sh
+
+PROG=hyperfine
+VER=1.10.0
+PKG=ooce/util/hyperfine
+SUMMARY="benchmarking tool"
+DESC="$PROG is a command-line benchmarking tool that provides statistics about command execution time."
+
+BUILD_DEPENDS_IPS=ooce/developer/rust
+
+set_arch 64
+set_checksum none
+
+SKIP_LICENCES=Various
+
+build() {
+    logmsg "Building 64-bit"
+    pushd $TMPDIR/$BUILDDIR >/dev/null
+    args="--release"
+    logcmd cargo build $args || logerr "build failed"
+    popd >/dev/null
+}
+
+install() {
+    logmsg "Installing"
+    pushd $TMPDIR/$BUILDDIR >/dev/null
+
+    logcmd mkdir -p $DESTDIR/$PREFIX/bin
+    logcmd cp target/release/$PROG $DESTDIR/$PREFIX/bin/fd || logerr "cp failed"
```

Comment:
The binary name needs adjusting here.
```suggestion
    logcmd cp target/release/$PROG $DESTDIR/$PREFIX/bin/$PROG \
        || logerr "cp failed"
```

When I tried building this, I got:
```
ERROR pkglint.dupaction001.1      path opt/ooce/bin/fd is a duplicate delivered by pkg:/ooce/util/hyperfine@1.10.0,5.11-151034.0 pkg://extra.omnios/ooce/util/fd@7.3.0,5.11-151034.0:20200329T132943Z under all variant combinations
```